src/lib/receiver.js

```diff
@@ -18,6 +18,12 @@ const SSP_RECEIVER_ID_PREFIX = '~ssp.'
  */
 
 /**
+ * @callback reviewPaymentCallback
```

Comment:
Is there a different way to express a function that returns a promise using JSdoc?